tests/Unit/TeamTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\Member;
+use App\Models\Team;
+use App\Models\User;
+use Tests\TestCase;
+
+class TeamTest extends TestCase
+{
+    /**
+     * A basic unit test example.
+     *
+     * @return void
+     */
+    public function test_createWithOwner()
+    {
+        $team_count = Team::count();
+        $member_count = Member::count();
+
+        $data = ['name' => 'teamname'];
+        $user = User::factory()->create();
+
+        $team = Team::createWithOwner($user, $data);
+        $member = $team->members()->where('user_id', $user->id)->first();
+
+        // Teams
+        $this->assertEquals($team_count + 1, Team::count());
+        $this->assertTrue($team->exists);
+        $this->assertEquals($team->name, $data['name']);
```

Comment:
```suggestion
        $this->assertEquals('teamname', $team->name);
```

assertEquals 書き順 にご注意です。
`assertEquals([期待される値], [実際の値])` なので、他の箇所も順序が逆になっているようですね。直しましょう。
また、研修の事前課題の自動テストのドキュメントでもやりましたが、テストで利用する値はハードコーディングが基本です。

Reply:
@ms2sato 
ご指摘ありがとうございます！
[454d32a](https://github.com/yuto14prog/ul-traning-202405/pull/7/commits/454d32a4e6af125deedf2d8b6855afa97c65efe1)で修正いたしました